### ISSUE TITLE
Drake still claims to have an innate that makes him only killable with Stake in Parallel Universe

### ISSUE THREAD

**Issue by purple-pixie** [TerryCavanagh/diceydungeons.com#1471]
I imagine that the problem is the description and not him being killable without stakes given he gets Vampire? instead

**Comment by findersjeepers** [comment]
Duplicate of #1019

**Comment by purple-pixie** [comment]
I foolishly only searched Drake and not vampire
